articles/prevent-tampering-of-fleet-agent.md

```diff
@@ -0,0 +1,71 @@
+# Prevent tampering of Fleet agent and installer settings on Windows
+
+## Introduction
+On Windows, users with administrative rights can remove or modify management agents like **Fleet Orbit**.  
+Unlike many EDR and DLP tools with built-in tamper protection, Fleet doesn’t attempt to hide or lock itself down. This is intentional—Fleet is built on **openness and transparency**, with enforcement driven by policies you can see and manage.
+
+To keep the agent in place, IT teams can add their own guardrails using a combination of:
+
+- A PowerShell script that sets hardened registry values  
+- A Fleet policy that checks those values and reapplies them if missing  
+- A Windows configuration profile that blocks MDM unenrollment  
+
+Together, these components create a **self-healing enforcement loop** that ensures protections remain in place, without relying on hidden or opaque mechanisms.
+
+---
+
+## Hardening the installer
+Apply registry values via a PowerShell script to help prevent uninstallation or tampering of protected applications.
+
+[Windows hardening PowerShell script](https://github.com/fleetdm/fleet/blob/main/assets/scripts/windows-fleet-hardening.ps1)
+
+---
+
+## Policies in Fleet
+A Fleet policy confirms that the hardened registry key exists. If it’s missing, the policy fails, triggering automation to run the script again. Once applied, the policy becomes compliant. If tampering occurs later, the cycle repeats.
+
+[Fleet policy for Windows hardening](https://github.com/fleetdm/fleet/blob/main/assets/policies/windows-fleet-hardening.policies.yml)
+
+**Note:** On first run, this policy intentionally fails to ensure automation executes the hardening script.
+
+---
+
+## Blocking unenrollment
+Deploy a configuration profile to prevent devices from unenrolling from MDM.
+
+[Block MDM unenrollment configuration profile](https://github.com/fleetdm/fleet/blob/main/assets/configuration-profiles/BlockMDMUnenrollment.xml)
+
+**Microsoft CSP reference**  
+The `AllowManualMDMUnenrollment` CSP is only supported on certain versions of Windows—verify compatibility before deployment.
+
+---
+
+## More advanced approaches
+More mature organisations may already be using additional controls to protect against tampering. For example:
+
+- **ADMX-backed CSPs** such as `ADMX_AddRemovePrograms`, which can restrict software removal across the board.  
+  - Provides strong protection, but applies to *all* applications, not just the Fleet agent.  
```

Comment:
```suggestion
  - Protects all applications, not just Fleet. 
```

Reply:
Since this is in list format, I suggest tightening these to make them more scannable.